### PR TITLE
fragment_write_a_review.xml & WriteAReviewFragment.kt

### PR DESCRIPTION
**Pull request by Puh00**
Finished design for fragment_write_a_review.xml and its functions in WriteAReviewFragment.kt

### CODE REVIEW

**Comment by Puh00** [comment]
hmm

File: app/src/main/java/com/example/bookly/AddABookFragment.kt [hunk 1]
```diff
@@ -75,7 +75,6 @@ class AddABookFragment : Fragment() {
 
     private fun previousFragment(){
         this.findNavController().popBackStack()
-
```
Comment (kusuoki) [inline app/src/main/java/com/example/bookly/AddABookFragment.kt:78]: impressive algorithm right here

Comment (kusuoki) [reply app/src/main/java/com/example/bookly/AddABookFragment.kt:78]: its beautiful

File: app/src/main/java/com/example/bookly/WriteAReviewFragment.kt [hunk 2]
```diff
@@ -1,26 +1,84 @@
...
+    private fun initListeners() {
+        binding.saveReviewButton.setOnClickListener {
+            BooklyDataHandler.getInstance().addReview(
+                currentBook,
+                binding.ratingBar.rating,
+                binding.reviewEditText.text.toString(),
+                Date()
+            )
+        }
+    }
```
Comment (kusuoki) [inline app/src/main/java/com/example/bookly/WriteAReviewFragment.kt:83]: k

**Comment by kusuoki** [comment]
lgtm

**Pull request merged**